.taskcluster.yml

```
@@ -0,0 +1,26 @@
+version: 1
+policy:
+  pullRequests: public
+tasks:
+  $match:
+    '(tasks_for == "github-pull-request" && event["action"] in ["opened","reopened","synchronize"])':
+      taskId:
+        $eval: as_slugid("pr_task")
+      provisionerId: proj-getting-started
+      workerType: tutorial
```

You'll want
```suggestion
      provisionerId: proj-misc
      workerType: ci
```

The config for this deployment is in https://github.com/mozilla/community-tc-config/, which [allows all mozilla/* repos to use this worker pool](https://github.com/mozilla/community-tc-config/blob/24423f6b33f7988c6c06db5b861d3facedfbabd4/config/projects/misc.yml#L59-L63).  We happily take pull requests there to modify the config, but for the moment that shouldn't be necessary.

I know it's difficult to discover these things, and partly that's because it's specific to the deployment (this worker pool wouldn't work in the Firefox-CI Taskcluster deployment) and partly it's specific to the repo (e.g., `gijsk/readability` does not have permission to use this worker pool).